### PR TITLE
Remove legacy for certificate auth & use latest versions of dependencies

### LINKED ISSUE #58: Remove unnecessary middleware method to load client certificate from request header
## Problem
The `Startup` class still has a middleware method to load the client certificate from the `X-ARR-ClientCert` header. This will become unnecessary since the `CertificateAuthenticationFilter` will automatically load this in the future.

## Solution
This filter is part of the [arcus.webapi](https://github.com/arcus-azure/arcus.webapi) repo and will become available for all versions above **v.0.2.0**.

### PR DESCRIPTION
**Pull request by tomkerkhove**
- Remove legacy for certificate auth
- Use latest versions of dependencies, include Arcus Web API

Closes #58

### CODE REVIEW

File: src/Arcus.Templates.WebApi/Startup.cs [hunk 1]
```diff
@@ -123,46 +120,5 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             });
 //[#endif]
         }
-#if CertificateAuth
```
Comment (stijnmoreels) [inline src/Arcus.Templates.WebApi/Startup.cs]: Always fun to remove 🎉

Comment (tomkerkhove) [reply src/Arcus.Templates.WebApi/Startup.cs]: Yes it is! It's also fun to open a PR without running the tests so 🤞 😂

Comment (tomkerkhove) [reply src/Arcus.Templates.WebApi/Startup.cs]: Too bad, would've been too easy 😂

File: src/Arcus.Templates.WebApi/Arcus.Templates.WebApi.csproj [hunk 2]
```diff
@@ -56,7 +56,7 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.2.0" />
-    <PackageReference Include="Arcus.WebApi.Logging" Version="0.2" />
+    <PackageReference Include="Arcus.WebApi.Logging" Version="0.3.0" />
     <PackageReference Include="Arcus.WebApi.Security.Authentication" Version="0.2" Condition="'$(Auth)' == 'true'" />
```
Comment (stijnmoreels) [inline src/Arcus.Templates.WebApi/Arcus.Templates.WebApi.csproj]: This should also be updated ;) I think here's your problem with the build.

Comment (tomkerkhove) [reply src/Arcus.Templates.WebApi/Arcus.Templates.WebApi.csproj]: Ah it looks like the Package Manager doesn't like those conditions 🙄 
image

**Pull request merged**